.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -67,72 +71,69 @@ git push upstream [VERSION]-rc.[RC]
+Check for [new releases](https://github.com/PeterFeicht/cppreference-doc/releases) and update the link and hash in [test/documentation/sharg-doxygen.cmake](https://github.com/seqan/sharg-parser/blob/1.0.0/test/documentation/sharg-doxygen.cmake#L55).
 You can compute the hash via `wget -O- <link to html book> | sha256sum`.
 
 </details>
 <a name="freeze"></a>
 <details><summary>Freezing the release branch</summary><br>
 
 - Make sure all PRs that should be merged are merged.
-- Set `SEQAN3_RELEASE_CANDIDATE` to `0` [include/seqan3/version.hpp](https://github.com/seqan/seqan3/blob/3.0.2/include/seqan3/version.hpp#L19-L24).
+- Set `SHARG_RELEASE_CANDIDATE` to `0` [include/sharg/version.hpp](https://github.com/seqan/sharg-parser/blob/1.0.0/include/sharg/version.hpp#L24).
 - This should be the last commit before the release.
 
 </details>
 <a name="versioned-docs"></a>
 <details><summary>Creating versioned documentation</summary><br>
 
 1. Checkout the release tag and build documentation.
-2. Create a #.#.# directory for the release in `/web/docs.seqan.de/htdocs/seqan/`
+2. Create a #.#.# directory for the release in `/web/docs.seqan.de/htdocs/sharg/`
 3. Copy everything from the build (`doc_usr/html/*`) into the directory.
-4. Alter the file `/web/docs.seqan.de/htdocs/seqan3.html` with a link to the new documentation build.
+4. Alter the file `/web/docs.seqan.de/htdocs/sharg.html` with a link to the new documentation build.
 
 </details>
 <a name="packaging"></a>
 <details><summary>Creating source packages</summary><br>
 
 Use a new clone of the repository.
 ```bash
-git clone https://github.com/seqan/seqan3.git
-cd seqan3
+git clone https://github.com/seqan/sharg-parser.git
+cd sharg-parser
 git checkout release-[VERSION] # version/branch to pack
-git submodule update --init
 
 mkdir ../package-build
 cd ../package-build
 
-cmake ../seqan3 # configure
-cpack # builds binary package, e.g. seqan3-[VERSION]-Linux.tar.xz{,.sha265}
-cmake --build . --target package_source # builds source package, e.g. seqan3-[VERSION]-Source.tar.xz{,.sha265}
+cmake ../sharg-parser # configure
+cpack # builds binary package, e.g. sharg-parser-[VERSION]-Linux.tar.xz{,.sha265}
+cmake --build . --target package_source # builds source package, e.g. sharg-parser-[VERSION]-Source.tar.xz{,.sha265}
 ```
 
-Note: Do not use `git clone --recurse-submodules https://github.com/seqan/seqan3.git` because it will recursively pull sub-submodules!
-
 </details>
 <a name="changelog"></a>
 <details><summary>Checking the changelog</summary><br>
 
-- List all supported compiler, also add to http://docs.seqan.de/seqan/3-master-user/about_api.html#autotoc_md35.
-- Check that all links are consistent, e.g., `[\#2540](https://github.com/seqan/seqan3/pull/2538)`:
+- List all newly supported compilers or dropped compilers.
+- Check that all PR links are consistent, e.g., `[\#2](https://github.com/seqan/sharg-parser/pull/2)`:
```

Comment:
We could also add the general link-checking somewhere appropriate.

https://github.com/seqan/sharg-parser/blob/main/test/scripts/link_check.sh

Reply:
Did we even include PR links in the changelog for sharg?

Reply:
No, but it doesn't make sense for the first release anyway.